cmd/podman/common/volumes.go

```diff
@@ -209,36 +209,21 @@ func getBindMount(args []string) (spec.Mount, error) {
 		switch kv[0] {
 		case "bind-nonrecursive":
 			newMount.Options = append(newMount.Options, "bind")
-		case "readonly", "read-only":
-			if setRORW {
-				return newMount, errors.Wrapf(optionArgError, "cannot pass 'readonly', 'ro', or 'rw' options more than once")
-			}
-			setRORW = true
-			switch len(kv) {
-			case 1:
-				newMount.Options = append(newMount.Options, "ro")
-			case 2:
-				switch strings.ToLower(kv[1]) {
-				case "true":
-					newMount.Options = append(newMount.Options, "ro")
-				case "false":
-					// RW is default, so do nothing
-				default:
-					return newMount, errors.Wrapf(optionArgError, "readonly must be set to true or false, instead received %q", kv[1])
-				}
-			default:
-				return newMount, errors.Wrapf(optionArgError, "badly formatted option %q", val)
-			}
-		case "ro", "rw":
+		case "readonly", "ro", "rw":
```

Comment:
doesn't `read-only` still need to be here?

Reply:
Tom, thanks for asking.

`read-only` was in the manual, but this option had never been implemented in the code before. 

mheon replaced `read-only` with `readonly` in the manual (https://github.com/containers/libpod/pull/6380/), I went ahead and removed `read-only` from the code. As the final result, the `readonly` is in the manual and code, the same behavior as Docker.

Have a nice day.
Charles